bot/resources/tags/docstring.md

```diff
@@ -1,18 +1,19 @@
-A [`docstring`](https://docs.python.org/3/glossary.html#term-docstring) is a string with triple quotes that often used in file, classes, functions, etc. A docstring should have a clear explanation of exactly what the function does. You can also include descriptions of the function's parameter(s) and its return type, as shown below.
+A [`docstring`](https://docs.python.org/3/glossary.html#term-docstring) is a string with triple quotes that's often used in files, classes, functions, etc. A docstring should contain a clear explanation of exactly what the function does. You can also include descriptions of the function's parameter(s) and its return type, as shown below.
 ```py
 def greet(name, age) -> str:
-  """
-  Return a string that greets the given person, including their name and age.
+    """
```

Comment:
Doesn't PEP 257 say docstrings should be written like this:
```py
"""A magic method, we don't know what it does.

Only kept here for legacy reasons, if we remove it the code breaks.
"""
```

Reply:
I don't think it mandates that, it uses that in some examples, but then also uses the blank first line as an example lower down, saying they're equivalent. At least that's how I read the pep